plugins/com.google.cloud.tools.eclipse.appengine.validation/plugin.xml

```diff
@@ -15,6 +16,7 @@
       name="%appengine.web"
       point="org.eclipse.wst.validation.validatorV2" 
       id="appEngineWebXmlValidator">
+    <!-- TODO should this specify more than one marker type? -->
```

Comment:
I wonder if we could do away with the jdt marker.  When I experimented in setting the RuntimeProblem as being an ERROR, I'd see an overlapping warning and error sign in the editor's overlay ruler (the left-hand side).  I suspect we're not filling in a marker property used by the JDT marker, which is defaulting to WARNING.

Reply:
Maybe. This is the first time I've looked at this code since @abbiefarr merged it last year. I'm sure there are some improvements to be made. I know some of the thing sI saw when I dug in surprised me.